SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,21 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # GDK for Unreal directory structure
-The table below contains the contents of the built GDK for Unreal after running `BuildGDK.bat`.
+The table below contains the contents of the Unreal GDk repo folder after running `Setup.bat`.
```

Comment:
`UnrealGDK` repository folder

Reply:
"GDK for Unreal repository folder" or "GDK repository folder" (because we have the context from the title).

(we have had a stealth rebrand)

Reply:
I am aware of the rebrand, however the repository is still called `UnrealGDK`, and the folder will be called the same by default.

Reply:
"contains the contents"
should be
"lists the contents"
to avoid a confusing double use of contain/ents.

Reply:
Use "directory".